src/dynarec/rv64/dynarec_rv64_f0.c

```diff
@@ -339,6 +339,101 @@ uintptr_t dynarec64_F0(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int ni
                     DEFAULT;
             }
             break;
+        case 0x10:
+            INST_NAME("LOCK ADC Eb, Gb");
+            READFLAGS(X_CF);
+            SETFLAGS(X_ALL, SF_SET_PENDING);
+            nextop = F8;
+            GETGB(x2);
+            SMDMB();
+            if((nextop&0xC0)==0xC0) {
+                if(rex.rex) {
+                    wback = xRAX + (nextop&7) + (rex.b<<3);
+                    wb2 = 0;
+                } else {
+                    wback = (nextop&7);
+                    wb2 = (wback>>2);
+                    wback = xRAX+(wback&3);
+                }
+                SRLIW(x1, wback, wb2*8);
+                ANDI(x1, x1, 0xFF);
+                emit_adc8(dyn, ninst, x1, x2, x3, x4, x5);
+                SLLI(x1, x1, wb2*8);
+                MOV_U12(x3, 0xFF);
+                SLLI(x3, x3, wb2*8);
+                NOT(x3, x3);
+                AND(wback, wback, x3);
+                OR(wback, wback, x1);
+            } else {
+                addr = geted(dyn, addr, ninst, nextop, &wback, x1, x3, &fixedaddress, rex, LOCK_LOCK, 0, 0);
+                ANDI(x3, wback, 0b11);
+                MOV_U12(x4, 0xff);
+                BNEZ_MARK(x3);
+                MARKLOCK;
+                LR_W(x5, wback, 1, 1);
+                AND(x4, x5, x4);
+                ANDI(x5, x5, -256);
+                ADDW(x6, x4, x2);
+                ANDI(x9, xFlags, 1 << F_CF);
```

Comment:
Is would be more optimal to to pre-add the Carry flags to GD a,d so move
```
ANDI(x9, xFlags, 1 << F_CF);
ADDW(x6, x2, x9);
```
Out of the loops.
Thoses needs some comment to be sure what regs old what value, to ease the debugging.
Like, line 374 x4 that was holding 0xff is destroy and now have [wback]&x4 inside the loop, that seems suspicious.